lua/ulx/modules/sh/cfc_warn_after.lua

```diff
@@ -25,3 +25,16 @@ kick:addParam{ type = ULib.cmds.PlayerArg }
 kick:addParam{ type = ULib.cmds.StringArg, hint = "reason", ULib.cmds.optional, ULib.cmds.takeRestOfLine, completes = ulx.common_kick_reasons }
 kick:defaultAccess( ULib.ACCESS_ADMIN )
 kick:help( "Kicks target and warns them." )
+
+function cmd.warnBanId( calling_ply, steamid, minutes, reason )
+    ulx.banid( calling_ply, steamid, minutes, reason )
+    local command = string.format( 'awarn_warn \"%s" "%s (%s ban)" ', steamid, reason, ULib.secondsToStringTime( minutes * 60 ) )
```

Comment:
I like to keep doubles outsides, singles inside. This should work

```suggestion
    local command = string.format( "awarn_warn '%s' '%s (%s ban)' ", steamid, reason, ULib.secondsToStringTime( minutes * 60 ) )
```

Reply:
The other warns had the same style, should i just change those too?

Reply:
Eh, nah. Leave it if it's standard for the project